packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -19,6 +19,7 @@ export default function TemplatePartInnerBlocks( {
 			value={ blocks }
 			onInput={ onInput }
 			onChange={ onChange }
+			__experimentalTagName="div"
```

Comment:
Why was this necessary for the inner-blocks?

Reply:
It tells the inner blocks component to return the inner blocks directly as an array instead of putting them inside its own wrapper. So we get this:

```
<header>
  <innerBlock1 />
  <innerBlock2 />
</header>
```

and not this:
```
<header>
  <innerBlocksDiv>
    <innerBlock1 />
    <innerBlock2 />
  </innerBlocksDiv>
</header>
```

It's weird because it's just a boolean check and "div" doesn't actually do anything. But this seems to be the format used by the group block and others.

Reply:
> It's weird because it's just a boolean check and "div" doesn't actually do anything. But this seems to be the format used by the group block and others.

Hah, that is weird...